### DISCUSSION TITLE
CSS classes or selectors for the icons in notes

### DISCUSSION THREAD

**Discussion by Oddward** [FlorianWoelki/obsidian-icon-folder#349] [Q&A] [answered]
I'm trying to style icons in the content via a stylesheet, specifically in a header, but I can't find any documentation or info about how to select the iconize icons. I'm also only using mobile these days so I can't inspect the UI and stuff.

At the moment, I have to use `h1 > *` in the CSS for the icon in the header which isn't ideal.

Please lemme know in what form they're placed in the rendered markdown (`<svg>`, `<i>`, `.icon`?) 

Thanks

**Comment by FlorianWoelki** [comment, marked as answer]
Hey, generally everything that is added to the DOM is notated somehow. For the editor, all added icons have the class `.cm-iconize-icon`. Hope that helps!

**Reply by Oddward** [reply]
I see, so anything from community modules is noted in a format like `.cm-module`, ya? Didn't realize that. 
Thanks!

**Reply by FlorianWoelki** [reply]
No worries :) 
No. It always depends on whether the developer added that class. For Iconize, this is the case.

**Reply by Oddward** [reply]
Is it exactly `.cm-iconize-icon` for every item or does it depend on the icon name/source? I can't seem to select child icons with that class name

**Reply by FlorianWoelki** [reply]
For icons in the editor it's always `.cm-iconize-icon`. For instance, if I apply the following `h1 > .cm-iconize-icon { color: green }`, it will apply the color green to all icons inside a `h1`.

**Reply by Oddward** [reply]
I updated the plugin the other day and then it worked lol. I had installed it not that long ago so I hadn't checked for updates
